doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3300,11 +3300,12 @@ the conversion in hypotheses :n:`{+ @ident}`.
    an expression defining a notation (e.g. `"_ + _"`), and this notation refers to an unfoldable constant, then the
    tactic unfolds it.
 
-.. tacv:: unfold @string%key
+.. tacv:: unfold @string%@scope
```

Comment:
Actually, what is designated by the `scope` token is the full name of the scope, and not the delimiting key. See https://coq.github.io/doc/master/refman/user-extensions/syntax-extensions.html#coq:cmd.delimit-scope. So this should rather be `ident`, unless we define a new token for delimiting scope keys.

Reply:
Good point.  I've put `ident` everywhere.